packages/playgrounds/src/common/client.js

```diff
@@ -13,7 +13,8 @@ const getClient = customOptions => {
           'io.cozy.konnectors',
           'io.cozy.procedures.administratives',
           'io.cozy.jobs:GET',
-          'io.cozy.jobs:POST:zip:worker'
+          'io.cozy.jobs:POST:zip:worker',
+          'io.cozy.bank.accounts.stats'
```

Comment:
Does this permission already added to banks / af ?

Reply:
Yes. But I forgot to add it to offline doctypes.